media-libs/libpulse/files/pulseaudio-16.0-fix-pactl-volume-command.patch

```diff
@@ -0,0 +1,53 @@
+From 05c06afa58e30b7958e96766d1e917099c8a4041 Mon Sep 17 00:00:00 2001
```

Comment:
Nit: Could you add a link to the relevant commit at the top of each patch?

Reply:
done now!